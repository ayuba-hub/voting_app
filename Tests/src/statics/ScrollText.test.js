import {shallow} from 'enzyme';
import React from 'react';
import ScrollText from './ScrollText';

describe("Test On ScrollText.js", ()=> {
    
    it('expects to render ScrollText component', () => {
        expect(shallow(<ScrollText/>).length).toEqual(1)
    })
    
    
    it('expects to render ScrollText component', () => {
        expect(shallow(<ScrollText/>)).toMatchSnapshot();
    })
    
})
