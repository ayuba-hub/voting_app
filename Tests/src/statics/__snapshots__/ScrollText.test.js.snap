// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test On ScrollText.js expects to render ScrollText component 1`] = `ShallowWrapper {}`;
